core/snippets/snippets/pythonComprehension.snippet

```diff
@@ -0,0 +1,23 @@
+language: python
+---
+
+name: listComprehension
+phrase: list comp
+insertionScope: statement
+-
+[$0 for $0 in $1]
+---
+
+name: setComprehension
+phrase: set comp
+insertionScope: statement
+-
+{$0 for $0 in $1}
+---
+
+name: dictComprehension
+phrase: dict comp
+insertionScope: statement
+-
+{$0: $2 for $2 in $1}
```

Comment:
In all three cases the first element to fill out in the snippet is the thing being iterated on, as it enables the IDE to help in filling out the other parts through suggestions.